examples/positron-static/src/positron/app.py

```diff
@@ -9,7 +9,7 @@ def translate_path(self, path):
         return str(self.server.base_path / path[1:])
 
 
-class LocalHTTPServer(HTTPServer):
+class LocalHTTPServer(ThreadingHTTPServer):
```

Comment:
Was this a required change, or just one that seemed like a good idea?

Reply:
The commit has a bit of this information but this was behaving strangely on Windows. The first few times I tried to start it, it would log "Starting server..." but not start serving the static page. After messing around with it for a bit, it seemed that after waiting a while, it would start working. But even then, it would often hang when trying to shutdown the server.

This was a lot of what we encountered when we added the web server for `briefcase run web`. Basically, it was a mess on Windows until we started using the `ThreadingHTTPServer`.

Reply:
Ah - I missed that the explanatory detail was in the commit message.